commit da9fd7839c9affea48f74a159a789fbb183b4be1
Author: maxice8 <30738253+maxice8@users.noreply.github.com>
Date:   Fri Feb 1 01:58:08 2019 -0200

    add missing stddef.h include for 'NULL' (#322)
    
    clientid.c: In function 'mailimap_clientid':
    clientid.c:66:38: error: 'NULL' undeclared (first use in this function)
       if (mailimap_read_line(session) == NULL)
                                          ^~~~
    clientid.c:66:38: note: 'NULL' is defined in header '<stddef.h>'; did you forget to '#include <stddef.h>'?
    clientid.c:39:1:
    +#include <stddef.h>
    
    clientid.c:66:38:
       if (mailimap_read_line(session) == NULL)

diff --git a/src/low-level/imap/clientid.c b/src/low-level/imap/clientid.c
index 1c34637..38880dd 100644
--- a/src/low-level/imap/clientid.c
+++ b/src/low-level/imap/clientid.c
@@ -33,6 +33,8 @@
 #	include <config.h>
 #endif
 
+#include <stdlib.h>
+
 #include "mailimap_sender.h"
 #include "clientid_sender.h"
 #include "clientid.h"
